docs/faqs.md

```diff
@@ -122,3 +122,29 @@ There are some recipes in `conan-center-index` that provide packages that contai
 We decided that these packages (as long as they match the premises) should list all the settings needed to build, so building from sources will generate the expected binary, but they will **remove `compiler` setting inside the `package_id()` method**. As a consequence, the CI will generate packages only for one compiler reducing the workload in the pipeline and the number of possible package IDs.
 
 Note about `build_type`.- We retain the `build_type` setting to make it possible for the users to _debug_ these installer packages. We considered removing this settings and it would be possible to compile these packages in _debug_ mode, but if we remove it from the packageID, the compiled package would override the existing _release_ binary, and it'd be quite inconvenient for the users to compile the binary every time they need to switch from _debug_ to _release_.
+
+## Can I remove an option from recipe
+
+No. Changing any option will result in a different package ID and may result a different behavior, the result can break users.
```

Comment:
please remove that argument, as adding `del self.info.options.foobar` causes the same effect as removing option (different package ID), so there is no advantage for that case